Akilahinternees/Andelachallenge
# ft(challenge2):User can create account[#168924993]

#### What does this PR do?
Have an end point that allow user to create account
#### Description of Task to be completed?
Have completed the following:

- POST /api/v1/auth/userSignup

#### How should this be manually tested?
After cloning the repo then run npm run start
and use postman to test that end point to see if it meet expectation
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#168924993
#### Screenshots (if appropriate)
![CREATEOOOO](https://user-images.githubusercontent.com/52999101/66162147-a1b6eb80-e5e2-11e9-87f4-aac83093ca70.JPG)

#### Questions:
N/A
